browser/css/jsdialogs.css

```diff
@@ -202,6 +206,10 @@ td.jsdialog > [id^='table-box']:not(.sidebar) {
 	overflow: auto;
 	max-height: 500px;
 	max-width: 700px;
+}
+/* - Also used in SpellingDialog where suggestion come in form
+     of an image*/
+#SpellingDialog-dialog-vbox .jsdialog.ui-scrollwindow {
```

Comment:
This width: max-content is very important for all places where image is inside scroll window.
Now in places like: Writer -> Character dialog, preview on the bottom is cropped. 
Or Format -> Theme icon view.

It was introduced to fix exactly that bug.
Please do opposite: keep it as is, but change value to initial for sort dialog (as it is only example where it breaks anything).

Reply:
really? the commit from Darshan doesn't say nothing related to that, Just a subject and just about one single dialog.

Reply:
commit doesn't say but it's true :)